### PR TITLE
Cost widget

### PR DESCRIPTION
**Pull request by ghost**
if you get data from:
```
voyages: hfocost & lsfocost
vessels_reports: hfo_consumption & lsfo_consumption
suggested_routes: total_cost
```
you will get the Remaining Cost

### CODE REVIEW

**Review by Swapna-Sahu — DISMISSED** [review]
Looks good to me

File: src/client/components/CostWidget.js [hunk 1]
```diff
@@ -3,23 +3,34 @@ import DataCard from "./DataCard";
...
 }
 
 CostWidget.propTypes = {
-  currentCost: PropTypes.number,
-  remainingCost: PropTypes.number
+  totalCost: PropTypes.number,
+  hfoCost: PropTypes.number,
+  lsfoCost: PropTypes.number,
+  hfoConsumption: PropTypes.number,
+  lsfoConsumption: PropTypes.number
```
Comment (zkwsk) [inline src/client/components/CostWidget.js]: You probably want to declare all of the props as required.

**Review by zkwsk — CHANGES_REQUESTED** [review]
Looks good :)

Can you please do the following:

- [x] Make proptypes required

- [x] Remove the category "Cost Widget" from Storybook and put it under the category "Cards".

File: src/client/components/CostWidget.js (hunk 1 above)
Comment (ghost) [reply src/client/components/CostWidget.js]: Done :)

**Review by zkwsk — APPROVED** [review]

**Pull request merged**